src/UI/Components/Diagram/styles.ts

```diff
@@ -102,6 +102,10 @@ export const CanvasWrapper = styled.div`
     stroke: var(--pf-v5-global--palette--light-green-300);
     fill: var(--pf-v5-global--palette--light-green-300);
   }
+  .joint-loose_element-highlight {
+    stroke: var(--pf-v5-global--palette--red-100);
```

Comment:
you picked the stronger ones for the green, perhaps that's why it appears to look saturated or muddy for the red.

Reply:
changed and made border smaller 
![Uploading Screenshot 2024-02-27 at 19.38.22.png…]()